test/e2e/suite/issuers/vault/certificate/approle_custom_mount.go

```diff
@@ -18,6 +18,7 @@ package certificate
 
 import (
 	"context"
+	"github.com/jetstack/cert-manager/test/e2e/framework/helper/featureset"
```

Comment:
as above, this import is in the wrong place; see our new [Coding Conventions](https://cert-manager.io/docs/contributing/coding-conventions/) for a description

Reply:
Fixed